unCLIP variant (#2297)
huggingface/diffusers: docs/source/en/api/pipelines/stable_unclip.mdx

<!--Copyright 2022 The HuggingFace Team. All rights reserved.

Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
the License. You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
specific language governing permissions and limitations under the License.
-->

# Stable unCLIP

> The stable unCLIP model is [stable diffusion 2.1](./stable_diffusion_2) finetuned to condition on CLIP image embeddings. 

Reviewer: ```suggestion
Stable unCLIP checkpoints are finetuned from [stable diffusion 2.1](./stable_diffusion_2) checkpoints to condition on CLIP image embeddings.
```

Would also be cool to mention later below which checkpoint was derived from which other checkpoint exactly
